platform-core-module/src/main/java/com/softicar/platform/core/module/environment/LiveSystemRevision.java

```diff
@@ -37,23 +38,36 @@ public Optional<String> getName() {
 		return Optional.ofNullable(name);
 	}
 
+	/**
+	 * Returns an Optional with the version of the current revision, or an empty
+	 * Optional if the revision version could not be determined.
+	 *
+	 * @return an Optional with the version of the current revision, or an empty
+	 *         Optional
+	 */
```

Comment:
when referring to other classes literally we usually link them, should be fixed on the `getName` method, too, IMO
```suggestion
	/**
	 * Returns an {@link Optional} with the version of the current revision, or an empty
	 * {@link Optional} if the revision version could not be determined.
	 *
	 * @return the optional version of the current revision
	 */
```